src/utils/logger.ts

```diff
@@ -6,7 +6,7 @@ export function logWarn(message: string) {
   if (window.console && window.console.warn) window.console.warn(message)
 }
 
-export function logError(messageOrException: string | Error, exception?: Error) {
+export function logError(messageOrException: unknown, exception?: unknown) {
```

Comment:
Ici on pourrait peut-être garder la `string` pour le message et juste typer unknown pour l'erreur, mais enfin je ne sais pas si ça a du sens d'avoir `string | unknown`
```suggestion
export function logError(messageOrException: string | unknown, exception?: unknown) {
```

Reply:
Effectivement ça n'a pas de sens d'avoir `string | unknown`.
J'ai passé `messageOrException` à `unknown` car la signature de `console.error` qui reprend ce paramètre est `any`. Or comme c'est mal d'utiliser `any` le mieux c'est d'utiliser `unknown` :)